frontend/lib/src/dataframes/Quiver.test.ts

```diff
@@ -691,8 +691,8 @@ describe("Quiver", () => {
         })
       })
 
-      test("datetime", () => {
```

Comment:
Is there a reason why this is being deleted in the diff? I would expect just simply addition.

Reply:
Before there was a mock DATETIME, but it really was just dates, so I properly labeled it as DATE and created the official DATETIME and DATETIMETZ.

This file I just changed so that it's using the correct MOCK data.